packages/web-components/src/search/search.styles.ts

```diff
@@ -0,0 +1,183 @@
+import { css, ElementStyles } from '@microsoft/fast-element';
+import {
+  Button,
+  DesignToken,
+  display,
+  ElementDefinitionContext,
+  forcedColorsStylesheetBehavior,
+  FoundationElementDefinition,
+} from '@microsoft/fast-foundation';
+import {
+  heightNumber,
+  inputFilledForcedColorStyles,
+  inputFilledStyles,
+  inputForcedColorStyles,
+  inputStateStyles,
+  inputStyles,
+} from '../styles';
+import { appearanceBehavior } from '../utilities/behaviors';
+import { bodyFont, controlCornerRadius, density, designUnit, neutralFillInputRecipe, neutralFillStealthRecipe, neutralForegroundRest, typeRampBaseFontSize, typeRampBaseLineHeight } from '../design-tokens';
+import { DirectionalStyleSheetBehavior } from '../styles';
+import { Swatch } from '../color/swatch';
+
+/**
+ * LTR styles for calendar
+ * @internal
+ */
+ const ltrStyles = css`
```

Comment:
I think you don't need the ltr/rtl now that the absolute position is gone, but I wonder if they layout is correct in rtl. Maybe 1px margin all around the clear button instead of this right or left positioning.